src/polycubed/src/base_cube.cpp

```diff
@@ -363,11 +365,20 @@ LogLevel BaseCube::get_log_level() const {
   return level_;
 }
 
-json BaseCube::to_json() const {
-  json j = {
-      {"name", name_}, {"uuid", uuid_.str()}, {"service", service_name_},
-      //{"type", type_},
-  };
+void BaseCube::set_conf(const nlohmann::json &conf) {
+  if (conf.count("loglevel")) {
```

Comment:
Should we use a #define for loglevel?

Reply:
I don't get the point, could you please clarify?

Reply:
Hello Mauricio. Just to avoid plain strings in code, if possible, e.g.,
"loglevel"
==> 
#define LOGLEVEL_STRING "loglevel"
.... if (conf.count(LOGLEVEL_STRING(( {...

Reply:
Yes, that can be done, but in this case I cannot see any advantage, the "loglevel" reference is tight coupled to the code, so in the remote case we change that we should also change methods, variables and so on.

Reply:
got it, thanks!